### PR TITLE
vim-patch:8.1.013{0,1}

### PR DESCRIPTION
**Pull request by blueyed**
patch 8.1.0131: :profdel is not tested

Problem:    :profdel is not tested.
Solution:   Add a test. (Dominique Pelle, closes vim/vim#3123)
https://github.com/vim/vim/commit/1fbfe7c48cb711f5a6deae535b3ec3bfe7952ce9

### CODE REVIEW

**Comment by justinmk** [comment]
Test is failing, does it pass locally? You can [run the test](https://github.com/neovim/neovim/blob/master/test/README.md#running-tests) like this:

    TEST_FILE=test_profile.res make oldtest

```
From test_profile.vim:
Found errors in Test_profdel_func():
function RunTheTest[35]..Test_profdel_func line 35: Expected 'Called 1 time' but got 'Called 2 times'
Found errors in Test_profdel_star():
function RunTheTest[35]..Test_profdel_star line 19: Expected 'Called 1 time' but got 'Called 2 times'
```

**Comment by blueyed** [comment]
It was missing the actual fix (previous patch).

(I am working based on patches that apply cleanly, in this case I did not really pay attention)

File: src/nvim/eval.c [hunk 1]
```diff
@@ -21628,25 +21624,28 @@ static void func_do_profile(ufunc_T *fp)
...
 
+  if (!fp->uf_prof_initialized) {
   if (len == 0)
     len = 1;      /* avoid getting error for allocating zero bytes */
-  fp->uf_tm_count = 0;
-  fp->uf_tm_self = profile_zero();
-  fp->uf_tm_total = profile_zero();
+    fp->uf_tm_count = 0;
+    fp->uf_tm_self = profile_zero();
+    fp->uf_tm_total = profile_zero();
```
Comment (justinmk) [inline src/nvim/eval.c]: indent does not match line 21628

Comment (blueyed) [reply src/nvim/eval.c]: Thanks, it was actually completely broken.

**Pull request merged**